.github/workflows/link-check-deploy.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   run:
     name: Initialize
     runs-on: ubuntu-latest
-    if: github.event.deployment_status.state == 'success'
+    if: github.ref != 'master' && github.event.deployment_status.state == 'success'
```

Comment:
should we remove ignore branch?

Reply:
It is removed, I assume it looks different on the default view because I reused the old branch

See the commit that removes it [here](https://github.com/iterative/dvc.org/pull/1795/commits/5e7889a3b58877a8bbe4f386f3866011e89a8843)

Reply:
I merged it and it hasn't been not removed.

Reply:
Sorry about that, I guess I have to merge master back when re-using a branch and the file changes tab was more trustworthy than I thought